src/main/java/blusunrize/immersiveengineering/common/items/RailgunProjectiles.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * BluSunrize
+ * Copyright (c) 2020
+ *
+ * This code is licensed under "Blu's License of Common Sense"
+ * Details can be found in the license file in the root folder of this project
+ */
+
+package blusunrize.immersiveengineering.common.items;
+
+import blusunrize.immersiveengineering.api.IETags;
+import blusunrize.immersiveengineering.api.tool.RailgunHandler;
+import blusunrize.immersiveengineering.api.tool.RailgunHandler.RailgunRenderColors;
+import blusunrize.immersiveengineering.common.entities.SawbladeEntity;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.projectile.AbstractArrowEntity;
+import net.minecraft.entity.projectile.TridentEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.crafting.Ingredient;
+import net.minecraft.util.math.EntityRayTraceResult;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+import net.minecraftforge.common.Tags;
+
+import javax.annotation.Nullable;
+import java.util.UUID;
+
+public class RailgunProjectiles
+{
+	public static void register()
+	{
+		// Iron
+		RailgunHandler.registerStandardProjectile(Ingredient.fromTag(IETags.ironRod), 16, 1.25).setColourMap(
```

Comment:
Lack of consistency in spelling of Color/Colour

Reply:
>_>
<_<